GVFS/GVFS.UnitTests/Virtualization/FileSystemCallbacksTests.cs

```diff
@@ -69,11 +74,15 @@ public void BackgroundOperationCountMatchesBackgroundFileSystemTaskRunner()
                 backgroundTaskRunner.Count.ShouldEqual(2);
                 fileSystemCallbacks.BackgroundOperationCount.ShouldEqual(backgroundTaskRunner.Count);
             }
+
+            mockPlaceholderDb.VerifyAll();
         }
 
         [TestCase]
         public void GetMetadataForHeartBeatDoesNotChangeEventLevelWhenNoPlaceholderHaveBeenCreated()
         {
+            Mock<IPlaceholderCollection> mockPlaceholderDb = new Mock<IPlaceholderCollection>(MockBehavior.Strict);
+            mockPlaceholderDb.Setup(x => x.GetCount()).Returns(0);

```

Comment:
In the other tests you set `GetCount` to return 1 rather than 0.  Was there any significance to using 0 here instead of 1?

Reply:
To match the check below
`metadata.ShouldContain("PlaceholderCount", 0);`